appserver/monitoring-console/webapp/src/main/java/fish/payara/monitoring/web/ApiRequests.java

```diff
@@ -50,6 +50,7 @@
  * 
  * @author Jan Bernitt
  */
+@SuppressWarnings("squid:S1104")
```

Comment:
NB. This stops sonar from complaining about non private fields that are not final which is needed for JSON IO.